src/main/java/org/fundacionjala/coding/Sort.java

```diff
@@ -0,0 +1,47 @@
+package org.fundacionjala.coding;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+
+/**
+ *
+ * @author admin-hp
+ */
+
+public class Sort {
+
+    /**
+     *
+     * @param totalText cadena que ingresa para que sea ordenado.
+     * @return Retorna en string que ya fue modificado.
+     */
+
+    public String sortTheInnerContent(String totalText) {
+
+        ArrayList<String> resWords = new ArrayList<>();
+
+        for (String i : totalText.split(" ")) {
+
+            String pre = i.substring(0, 1);
+
+            String end = i.substring(i.length() - 1, i.length());
+
+            String body = i.substring(1, i.length() - 1);
+
+            String[] body2 = body.split("");
+
+            Arrays.sort(body2, Collections.reverseOrder(String.CASE_INSENSITIVE_ORDER));
+
+            body = String.join("", body2);
+
+            resWords.add(pre + body + end);
+
+        }
+
+        String resp = String.join(" ", resWords);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "resp". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)